client/src/components/Statistics.js

```diff
@@ -19,39 +20,36 @@ const Statistics = () => {
 
   return (
     <div
-      className="relative bg-gray-800 py-16"
-      style={{
-        backgroundImage: `url("https://www.toptal.com/designers/subtlepatterns/patterns/double-bubble-dark.png")`,
-      }}
+      className="relative bg-slate-800 py-2"
     >
       {/* Dark overlay */}
-      <div className="absolute inset-0 bg-black opacity-40"></div>
+      {/* <div className="dark-overlay absolute inset-0 bg-black opacity-40"></div> */}
 
       <div className="relative z-10 p-8">
         <div className="grid grid-cols-1 sm:grid-cols-2 md:grid-cols-3 lg:grid-cols-6 gap-6 text-white">
           <div className="bg-white bg-opacity-20 backdrop-filter backdrop-blur-lg p-6 text-center shadow-lg rounded-lg transform hover:scale-105 transition duration-300">
             <h1 className="text-5xl font-bold">{statistics.research_centres}</h1>
-            <p className="text-lg">Research Centres</p>
+            <p className="text-sm font-light  mt-3">Research Centres</p>
           </div>
-          <div className="bg-white bg-opacity-20 backdrop-filter backdrop-blur-lg p-6 text-center shadow-lg rounded-lg transform hover:scale-105 transition duration-300">
-            <h1 className="text-5xl font-bold">{statistics.rd_grants}</h1>
-            <p className="text-lg">R&D Grants</p>
+          <div className="content-around bg-white bg-opacity-20 backdrop-filter backdrop-blur-lg p-6 text-center shadow-lg rounded-lg transform hover:scale-105 transition duration-300">
+            <h1 className="text-5xl font-bold">{statistics.rd_grants}Cr+</h1>
```

Comment:
Remove the cr+ it will be easier for management later, in case they decide to change K to M and so on.

Reply:
Ok. This issue has been fixed in the latest commit.